src/blueprints/tv2/action-factories/tv2-remote-action-factory.ts

```diff
@@ -150,6 +151,34 @@ export class Tv2RemoteActionFactory {
     }
   }
 
+  private createInsertRemoteAsOnAirActions(blueprintConfiguration: Tv2BlueprintConfiguration): Tv2RemoteAction[] {
+    return [
+      ...blueprintConfiguration.studio.remoteSources,
+      ...blueprintConfiguration.studio.feedSources
+    ].map(source => this.createInsertRemoteAsOnAirAction(blueprintConfiguration, source))
+  }
+
+  private createInsertRemoteAsOnAirAction(blueprintConfiguration: Tv2BlueprintConfiguration, remoteSource: Tv2SourceMappingWithSound): Tv2RemoteAction {
+    const partId: string = `remoteInsertActionPart_${remoteSource.name}`.replaceAll(' ', '')
```

Comment:
Just want to point out that we are using `replaceAll(' ', '')` for ensuring correct ids in 10 places right now. Perhaps we should bring these together?

Reply:
Agree. Would be nice to do something about it